crates/core/tedge/src/cli/certificate/create.rs

```diff
@@ -10,27 +9,24 @@ use certificate::PemCertificate;
 use std::fs::File;
 use std::fs::OpenOptions;
 use std::io::prelude::*;
-use std::io::ErrorKind;
 use std::path::Path;
-use tedge_utils::file::create_file_with_mode_or_overwrite;
 use tedge_utils::paths::set_permission;
 use tedge_utils::paths::validate_parent_dir_exists;
-/// Create a self-signed device certificate
+
+/// Create self-signed device certificate and signing request
 pub struct CreateCertCmd {
     /// The device identifier
     pub id: String,
 
-    /// The path where the device certificate will be stored
+    /// The path where the device certificate / request will be stored
     pub cert_path: Utf8PathBuf,
 
     /// The path where the device private key will be stored
     pub key_path: Utf8PathBuf,
 
-    /// The path where the device CSR file will be stored
-    pub csr_path: Option<Utf8PathBuf>,
-
-    /// The component that is configured to host the MQTT bridge logic
-    pub bridge_location: BridgeLocation,
+    /// The owner of the private key
+    pub user: String,
+    pub group: String,
 }
```

Comment:
question: if `CreateCertCmd` creates certificate signing requests, what does the `CreateCsrCmd` do?

Is this a leftover of some previous changes, or am I not understanding something?

Reply:
Oops. This is a leftover. `CreateCertCmd` is now only used to create self signed certificate.

This now confusing comment has been added during an intermediate step and has to be removed.

Reply:
Fixed: https://github.com/thin-edge/thin-edge.io/pull/3268/commits/4cf7c8892af30efc438709975b548936314726e4